autopush/endpoint.py

```diff
@@ -17,73 +17,160 @@
 If no external routers are configured then the :class:`RegistrationHandler`
 will not be able to perform any additional router data registration.
 
-HTTP API
-========
+Push Service HTTP API
+=====================
+
+The following section describes how remote servers can send Push
+Notifications to apps running on remote User Agents.
 
 API methods requiring Authorization must provide the Authorization
 header containing the authrorization token. The Authorization token is returned
 as "secret" in the registration response.
```

Comment:
Well, this is the third auth schema we've implemented. "secret" is persisting here solely because it's what we used before and it's what we agreed on in the API 

If nothing else, you win for the finely crafted bikeshed emoji